landoapi/models/transplant.py

```diff
@@ -0,0 +1,87 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from landoapi.api.revisions import get_revision
+from landoapi.models.storage import db
+from landoapi.transplant_client import TransplantClient
+
+TRANSPLANT_JOB_STARTED = 0
+TRANSPLANT_JOB_FINISHED = 2
+
+
+class Transplant(db.Model):
+    __tablename__ = "transplants"
+
+    id = db.Column(db.Integer, primary_key=True)
+    request_id = db.Column(db.Integer, unique=True)
+    revision_id = db.Column(db.String(30))
+    status = db.Column(db.Integer)
```

Comment:
Personally I feel using a string here would be much better, e.g 'started', and 'finished'. 
In the API it is not clear what 0 or 2 means without looking at the source code, for an external client that is very confusing. In addition if we ever have to debug the database it is nice to see text of the statuses instead of a number.

What do you think?